packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1407,30 +1444,24 @@ class ImportedVpc extends VpcBase {
  * Returns "private subnets" by default.
  */
 function reifySelectionDefaults(placement: SubnetSelection): SubnetSelection {
-  if (placement.subnetType !== undefined && placement.subnetName !== undefined) {
+  if (placement.subnetName !== undefined) {
+    if (placement.subnetGroupName !== undefined) {
+      throw new Error(`Cannot supply both 'subnetName' and 'subnetGroupName', they mean the same thing. Prefer 'subnetGroupName'.`);
```

Comment:
It should read `Please use only "subnetGroupName" ("subnetName" is deprecated and has the same behavior)`